server_addon/maya/client/ayon_maya/plugins/publish/collect_yeti_rig.py

```diff
@@ -15,6 +15,12 @@
             "cbId"}
 
 
+ORNATRIX_NODES = {
+    "HairFromGuidesNode", "GuidesFromMeshNode",
+    "MeshFromStrandsNode", "SurfaceCombNode"
+}
```

Comment:
It's quite weird that now suddenly we have Ornatrix logic in a file related to Yeti? Seems wrong?

Reply:
The texture logics from yeti is also applicable for ornatrix too. maybe we can somehow implement the relevant logic in `lib.py` and use it here. (We can separate into two collectors).

Reply:
Yup - either make it abstract and reusable - or if it's still very specific to these two plug-ins maybe change just the filename. Up to you. :)